### ISSUE TITLE
Deploy on save, suppress notification

### ISSUE THREAD

**Issue by whydoidoit** [mkloubert/vs-deploy#13]
Firstly I really like the extension, helps a lot! However I'd really like to use deploy on save but I can't stand the constant interruption of popup messages telling me files have been sent to the server.  I looked for an option to hide this, or just have it as a status bar message but couldn't find one.  It's making the feature unusable for me, because it's so distracting.

**Comment by mkloubert** [comment]
Thanks! :-)

**I have implemented the feature since Version 3.26.0 and it is available now.**

You simply need to set the `showPopupOnSuccess` property to `(false)`.

```json
{
    "deploy": {
        "showPopupOnSuccess": false
    }
}
```

It works for any kind of deployment. Warnings and errors will still be displayed in popups (if occurring).

**Comment by enly1** [comment]
FYI - This works like a charm as well - been doing some stuff with it on and some with it off - working as expected.
